### PR TITLE
Add aerodrome to `dex.trades_beta`

### PR DESCRIPTION
**Pull request by tomfutago**
linked to #4759

### CODE REVIEW

**Comment by tomfutago** [comment]
Hey @Hosuke, this might be another edge case - it looks like aerodrome is a uniswap V2 fork (all columns match on `Pair_evt_Swap`), but factory pool event has 'pool' rather than 'pair' column (as in uni V3). I updated macro to add `pair_column_name` with default 'pair' so it's backward compatible and shouldn't affect existing spells. Only for cases like aerodrome I can pass 'pool' for that column name and still be able to use macro rather than copy the whole spell. If you disagree let me know.

On another note, I'm getting this error msg now - any hints what's that about?

`TrinoExternalError(type=EXTERNAL, name=DELTA_LAKE_INVALID_SCHEMA, message="Metadata not found in transaction log for test_data.sushiswap_celo_base_trades_seed"`

**Comment by Hosuke** [comment]
> Hey @Hosuke, this might be another edge case - it looks like aerodrome is a uniswap V2 fork (all columns match on `Pair_evt_Swap`), but factory pool event has 'pool' rather than 'pair' column (as in uni V3). I updated macro to add `pair_column_name` with default 'pair' so it's backward compatible and shouldn't affect existing spells. Only for cases like aerodrome I can pass 'pool' for that column name and still be able to use macro rather than copy the whole spell. If you disagree let me know.

I think we can extend the macro's compatibility. cc. @jeff-dude 


> On another note, I'm getting this error msg now - any hints what's that about?
> 
> `TrinoExternalError(type=EXTERNAL, name=DELTA_LAKE_INVALID_SCHEMA, message="Metadata not found in transaction log for test_data.sushiswap_celo_base_trades_seed"`

We may need to find a way to load the existing seed files when needed.

**Comment by jeff-dude** [comment]
> I think we can extend the macro's compatibility. cc. @jeff-dude

yes, i'm open to extending the macro to accept word switches on column / table / whatever is needed 👍 



> We may need to find a way to load the existing seed files when needed.

commented in sushiswap PR about why it's happening. for now, just flip a row with another to push into this PR and have CI build it fresh.

**Comment by Hosuke** [comment]
> Hey @Hosuke, this might be another edge case - it looks like aerodrome is a uniswap V2 fork (all columns match on `Pair_evt_Swap`), but factory pool event has 'pool' rather than 'pair' column (as in uni V3). I updated macro to add `pair_column_name` with default 'pair' so it's backward compatible and shouldn't affect existing spells. Only for cases like aerodrome I can pass 'pool' for that column name and still be able to use macro rather than copy the whole spell. If you disagree let me know.
> 
> On another note, I'm getting this error msg now - any hints what's that about?
> 
> `TrinoExternalError(type=EXTERNAL, name=DELTA_LAKE_INVALID_SCHEMA, message="Metadata not found in transaction log for test_data.sushiswap_celo_base_trades_seed"`

Solved. @tomfutago

File: macros/models/_sector/dex/uniswap_compatible_trades.sql [hunk 1]
```diff
@@ -4,6 +4,7 @@
     , version = null
     , Pair_evt_Swap = null
     , Factory_evt_PairCreated = null
+    , pair_column_name = 'pair'
```
Comment (Hosuke) [inline macros/models/_sector/dex/uniswap_compatible_trades.sql:7]: I think this implementation is simple and clean.

**Review by jeff-dude — APPROVED** [review]
thanks for keeping it clean with the macro enhancement 🤝

**Pull request merged**